test/interpolation.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Why is this file required? It looks like an example, a test required some expectations or asserts

Reply:
The issue asks for smoother movements which are brought about by Slerp under the threshold, while Squad works just fine above it. I don’t know how to assert if a given interpolation is smooth or not, so I added this file for anyone to check the movements themselves.